analysis/anx_dep_proms_cod_analysis.qmd

```diff
@@ -101,7 +119,8 @@ description_dict <- c(
   "Patient Health Questionnaire Nine Item score" = "PHQ-9 score",
   "Edinburgh postnatal depression scale score" = "EPDS score",
   "Assessment using Whooley depression screen" = "Whooley depression screening",
-  "Edinburgh postnatal depression scale" = "EPDS assessment"
+  "Edinburgh postnatal depression scale" = "EPDS assessment",
+  "Depression screening using questions" = "Depression screening"
```

Comment:
I decided to shorten the description for the table, do you think we should show the original description here? How about for all the other ones in the dict?